File: PHP/Program/TokoElektronik.php
<?php
require_once "Toko.php";

class TokoElektronik extends Toko {
    private $namaBarang, $idBarang, $hargaBarang, $fotoBarang;

    public function __construct($namaPerusahaan, $alamat, $tahunBerdiri,
                                $pemilik, $nib, $lokasi, $namaToko, $penghasilan, $jamOperasional,
                                $namaBarang, $idBarang, $hargaBarang, $fotoBarang) {
        parent::__construct($namaPerusahaan, $alamat, $tahunBerdiri,
                            $pemilik, $nib, $lokasi, $namaToko, $penghasilan, $jamOperasional);
        $this->setNamaBarang($namaBarang);
        $this->setIdBarang($idBarang);
        $this->setHargaBarang($hargaBarang);
        $this->setFotoBarang($fotoBarang);
    }

    // Setter
    public function setNamaBarang($v){ $this->namaBarang = $v; }
    public function setIdBarang($v){ $this->idBarang = $v; }
    public function setHargaBarang($v){ $this->hargaBarang = $v; }
    public function setFotoBarang($v){ $this->fotoBarang = $v; }

    // Getter
    public function getNamaBarang(){ return $this->namaBarang; }
    public function getIdBarang(){ return $this->idBarang; }
    public function getHargaBarang(){ return $this->hargaBarang; }
    public function getFotoBarang(){ return $this->fotoBarang; }
}

File: Java/TokoElektronik.java
import java.util.*;

public class TokoElektronik extends Toko {
    private String namaBarang;
    private String idBarang;
    private double hargaBarang;

    public TokoElektronik(String pemilik, String nib, String lokasi,
                          String namaToko, double penghasilan, String jamOperasional,
                          String namaBarang, String idBarang, double hargaBarang) {
        super(pemilik, nib, lokasi, namaToko, penghasilan, jamOperasional);
        this.namaBarang = namaBarang;
        this.idBarang = idBarang;
        this.hargaBarang = hargaBarang;
    }

    // Setter
    public void setNamaBarang(String v) { namaBarang = v; }
    public void setIdBarang(String v) { idBarang = v; }
    public void setHargaBarang(double v) { hargaBarang = v; }

    // Getter
    public String getNamaBarang() { return namaBarang; }
    public String getIdBarang() { return idBarang; }
    public double getHargaBarang() { return hargaBarang; }

    public List<String> getRow() {
        return Arrays.asList(
            getPemilik(), getNib(), getLokasi(), getNamaToko(),
            String.valueOf((long)getPenghasilan()), getJamOperasional(),
            getNamaBarang(), getIdBarang(), String.valueOf((long)getHargaBarang())
        );
    }

    public static List<String> getHeader() {
        return Arrays.asList("Pemilik","NIB","Lokasi","Nama Toko","Omset","Jam","Barang","ID","Harga");
    }
}
